lib/good_job/lockable.rb

```diff
@@ -143,7 +143,7 @@ def with_advisory_lock
       def supports_cte_materialization_specifiers?
         return @_supports_cte_materialization_specifiers if defined?(@_supports_cte_materialization_specifiers)
 
-        @_supports_cte_materialization_specifiers = ActiveRecord::Base.connection.postgresql_version >= 120000
+        @_supports_cte_materialization_specifiers = Job.connection.postgresql_version >= 120000
```

Comment:
The `GoodJob::Lockable` class is intended to be a generic ActiveRecord mixin, so this hopefully can be:

```suggestion
        @_supports_cte_materialization_specifiers = self.class.connection.postgresql_version >= 120000
```

Reply:
hmm, this doesn't work somehow

Reply:
Whoops. This is within `class_methods` so it just needs to be: `connection.postgresql_version >= 120000`